packages/react/src/components/AccountSettings/utils/displayText.ts

```diff
@@ -0,0 +1,18 @@
+export const defaultChangePasswordDisplayText = {
+  currentPasswordLabel: 'Current Password',
+  newPasswordLabel: 'New Password',
+  confirmPasswordLabel: 'Confirm Password',
```

Comment:
Not entirely sure what the convention is, for other components we use `text` and `buttonText` suffix, so in that regard `currentPasswordLabel` would make sense since it's the the text corresponding to the label. Perhaps we can standardize across all components that use displayText as followup?